illumos-utils/src/dladm.rs

```diff
@@ -6,12 +6,13 @@
 
 use crate::link::{Link, LinkKind};
 use crate::zone::IPADM;
```

Comment:
The changes in this file - and others within `illumos-utils/src/...` - are representative of "what actually is changing in this PR":

- std::process::Command -> tokio::process::Command
- This makes the function async
- The rest of the changes are fall-out from this, identifying blocks of code as asynchronous